.gitignore

```
@@ -58,3 +58,5 @@ override.tf.json
 terraform.rc
 
 !gradle/wrapper/gradle-wrapper.jar
+
+*/.DS_Store
```

```suggestion
.DS_Store
**/.DS_Store
```
https://git-scm.com/docs/gitignore
>A leading "**" followed by a slash means match in all directories. For example, "**/foo" matches file or directory "foo" anywhere, the same as pattern "foo".